### DISCUSSION TITLE
How to input flame detector?

### DISCUSSION THREAD

**Discussion by XYinlin** [firemodels/fds#11451] [General]
How to input flame detector to FDS?

**Comment by drjfloyd** [comment]
The detector's spefications or the specific listing test it was testeed against should give you some insight into what fire size you would expect to see at some distance. Usually distance is large enough compared to the fire size that you could treat the fire as a point source (a flame detector is looking for some strength of IR and/or UV light.). Based on where your detector will be in your model in relation to the fire, you could estimate what fire size you would expcet to detect and just set a DEVC that fire size as its setpoint. You could get a little more sophisticated and account for the detector's normal vector and the vector between the detector and the fire and adjust the HRR by 1/cos of the angle between the two.
